.ci/checker-framework.groovy

```diff
@@ -113,10 +113,16 @@ private static int checkCheckerFrameworkReport(final String profile) {
 private static List<List<String>> getCheckerFrameworkErrors(final String profile) {
     final List<String> checkerFrameworkLines = new ArrayList<>()
     final String command = "mvn -e --no-transfer-progress clean compile -P${profile}"
-    final Process process = getOsSpecificCmd(command).execute()
-    process.in.eachLine { final line ->
-        checkerFrameworkLines.add(line)
-        println(line)
+    final ProcessBuilder processBuilder = new ProcessBuilder(getOsSpecificCmd(command).split(' '))
+    processBuilder.redirectErrorStream(true)
```

Comment:
Will this obfuscate errors produced by maven during command execution?

Reply:
I don't think currently maven by default writes anything to `System.err`, all the info, error, and warning logs are written to `System.out`. 

See the [default SLF4J configuration for Maven ](https://github.com/apache/maven/blob/master/apache-maven/src/assembly/maven/conf/logging/simplelogger.properties). Output may go to `System.err` if changed to:
```properties
org.slf4j.simpleLogger.logFile=System.err
```

This makes me question whether this PR is required or not, but I will still go with this patch as this is a good practice, we should empty the error stream.

More details about maven logging at: https://maven.apache.org/maven-logging.html